docs/quickstart.rst

```diff
@@ -34,14 +34,18 @@ Using Web3
 ----------
 
 To use the web3 library you will need to initialize the
-:class:`~web3.Web3` class.
+:class:`~web3.Web3` class by connecting to an Ethereum node. The quickest way to do so for free is by setting up an account on `Infura https://infura.io/>`. On Infura, create a project and copy the Project ID. Then set the environment variable ``WEB3_INFURA_PROJECT_ID``.
+
+.. code-block:: shell
+    
+    $ export WEB3_INFURA_PROJECT_ID=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
 Use the ``auto`` module to :ref:`guess at common node connection options
-<automatic_provider_detection>`.
+<automatic_provider_detection>`. Or specify to use the Infura node by importing from ``web3.auto.infura``.
 
 .. code-block:: python
 
-    >>> from web3.auto import w3
```

Comment:
I think it would be clearer to keep this line, but maybe move it to after line 43. So something like: 

```
Use the ``auto`` module to :ref:`guess at common node connection options <automatic_provider_detection>`.

.. code-block:: python

    >>> from web3.auto import w3

Or, specify the use of an Infura node by importing from ``web3.auto.infura``:

.. code-block:: python

    >>> from web3.auto.infura import w3
    >>> w3.eth.blockNumber
    4000000
```

Reply:
I agree that would be more clear, but I'm trying to keep it as short as possible. Obviously using `eth.auto` was chosen for the quick start, however, I've found `eth.auto.infura` to "just work"  more reliably and I would even remove the `eth.atuo` part. Since `eth.auto` was chosen to use for a reason I think it's fine to leave in but the focus should be on getting it going fastest, not explaining various connecting possibilities, IMHO.  :)

What do you think?

Reply:
I agree that `web3.auto.infura` tends to work more reliably than `web3.auto` (since you have to either be running a node or have an environment variable set), and I think it adds value to have it in the Quickstart. Maybe we could add a note to the section or something: 

```
.. NOTE::

If importing your web3 instance via `web3.auto` doesn't work, try connecting to Infura using `web3.auto.infura`
```

Reply:
My point is that, since this is a "quick start" and not actually a tutorial `web3.auto.infura` should be default, and `web3.auto` should be foot note if that's how we want to do it

Reply:
I'm good with that. I just think it's confusing that it says use the web3 auto module and doesn't show how. We can add that piece to a note though.

Reply:
That's fair. moved the explanation of Auto and the link to other connection methods to a note under the connection block of code.